### DISCUSSION TITLE
Andare online con il nuovo server

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/opendatasicilia.it#35] [Idee/Proposte]
Ciao care/i,
abbiamo migrato i contenuti, fatto dei check, fatte delle modifiche e il sito "non nuovo", basato sul nuovo spazio di hosting è già navigabile seguendo [queste istruzioni](https://github.com/opendatasicilia/opendatasicilia.it/discussions/18).

Io, andrei online, in modo da rendere disponibile subito l'HTTPS, e basare eventuali nuovi post sul nuovo spazio/motore.
Il sito "veramente nuovo" è ancora in costruzione e si prenderà i suoi doverosi tempi.

Questa discussione per raccogliere quegli elementi bloccanti all'andare online, da trasformare così in issue da chiudere.
Li aggiungete come risposta qui?

Grazie

P.S. Grazie a @gbvitrano per il lavoro di ordine e struttura che ha già fatto sul nuovo spazio.

**Comment by gbvitrano** [comment]
in ritardo ma ci sono arrivato... :-) visto che momentaneamente va online il template che ho caricato, domani faccio altre piccole modifiche e poi lunedì può andare online.

Posso avere le credenziali ftp del nuovo spazio, vorrei modificare un file php del template, ma non è essenziale :-)

**Reply by aborruso** [reply]
> Posso avere le credenziali ftp del nuovo spazio, vorrei modificare un file php del template, ma non è essenziale :-)

Te le ho inviate, fammi sapere

**Reply by gbvitrano** [reply]
Ho configurato filezilla e la connessione viene rifiutata dal server
![image](https://user-images.githubusercontent.com/20488693/145713532-4d44c106-c0f6-4d13-b35f-a20d539554e5.png)

**Reply by aborruso** [reply]
ti ho inviato indirizzo server corretto.

Riprovi per favore?

**Comment by aborruso** [comment]
Ho aggiornato i DNS (vedi #44 )
